### PR TITLE
Fix pred results table for non-probabilistic LSGDClassifiers

### LINKED ISSUE #91: LinearSGDClassifier prediction results table rendering improperly

### PR DESCRIPTION
**Pull request by acrellin**
Closes #91

### CODE REVIEW

**Comment by bnaul** [comment]
Do the tests ever compute any types of model besides random forests? Maybe we should add an extra test in the model + prediction modules for another type of model (regressor and/or non-probabilistic classifier)?

**Comment by acrellin** [comment]
:+1: I'm on it

File: cesium_app/tests/fixtures.py
Comment (stefanv) [inline cesium_app/tests/fixtures.py]: Probably best not to use type, which is a Python built-in.  Also, can you add a little docstring here to say what "type" means?

File: cesium_app/tests/fixtures.py
Comment (stefanv) [inline cesium_app/tests/fixtures.py:46]: This may not be a safe location to write to on all systems, e.g. if cesium was installed as part of Debian or some other distribution where you don't have write access to the library location.

Comment (stefanv) [reply cesium_app/tests/fixtures.py:46]: (Use one of the temporary file utilities included in Python stdlib instead?)

File: cesium_app/tests/fixtures.py
Comment (stefanv) [inline cesium_app/tests/fixtures.py]: Add a short docstring

File: cesium_app/tests/frontend/test_build_model.py
Comment (stefanv) [inline cesium_app/tests/frontend/test_build_model.py]: There's lots of duplicated code between these functions---refactor to _private_function.

Comment (stefanv) [reply cesium_app/tests/frontend/test_build_model.py]: E.g.:

```
model = _select_model('RandomForestClassifier')
...
```

File: public/scripts/Predictions.jsx
Comment (stefanv) [inline public/scripts/Predictions.jsx]: On the Python side, this should probably be encoded explicitly.  E.g.

```
{'some_result':
    'prediction_has_probability': True,
    ...
}
```

**Review by stefanv — CHANGES_REQUESTED** [review]

File: cesium_app/tests/fixtures.py
Comment (acrellin) [reply cesium_app/tests/fixtures.py:46]: @stefanv not sure what you're talking about... No writing is happening here.

Comment (stefanv) [reply cesium_app/tests/fixtures.py:46]: Ah, perhaps I was misled by the function name: `create_test_dataset`!

**Comment by acrellin** [comment]
@stefanv implemented your suggestions, wanna take another look?

File: cesium_app/tests/fixtures.py
Comment (stefanv) [inline cesium_app/tests/fixtures.py:33]: This looks great.

File: cesium_app/tests/frontend/test_build_model.py
Comment (stefanv) [inline cesium_app/tests/frontend/test_build_model.py]: Perhaps call `str(id)` as the first line of `_build_model`, then these can disappear.

File: cesium_app/tests/frontend/test_build_model.py
Comment (stefanv) [inline cesium_app/tests/frontend/test_build_model.py]: Beautiful!

**Review by stefanv — APPROVED** [review]
This looks great, thanks @acrellin.  I have two minor suggestions, but :+1: after that (go ahead and merge if the tests pass).

**Pull request merged**
